src/test/auth.spec.js

```diff
@@ -0,0 +1,115 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+import TestHelper from '../utils/testHelper';
+import Helper from '../utils/helpers';
+
+chai.use(chaiHttp);
+chai.should();
+
+const URL_PREFIX = '/api/v1/auth';
+
+const apiEndpoint = '/api/v1/auth/login';
+
+const user = {
+  first_name: 'Samuel',
+  last_name: 'koroh',
+  email: 'user1@gmail.com',
+  password: '123456'
```

Comment:
Please update the password to `password: 'Ice5m5am0a843r03'` to enable test to pass